File: jarvis-mcp/tests/fixtures/configs.py
"""Configuration fixtures for tests."""

DEFAULT_CONFIG = {
    "language": "en",
    "response_style": "concise",
    "persona": "jarvis",
    "auto_capture": True,
    "checkpoint_before_risky": True,
}

VERBOSE_CONFIG = {
    **DEFAULT_CONFIG,
    "response_style": "verbose",
    "auto_capture": False,
}

MINIMAL_CONFIG = {
    "language": "en",
    "persona": "jarvis",
}

File: jarvis-cli/tests/fixtures/configs.ts
/**
 * Test fixtures for configuration objects
 */

export const mockGlobalConfig = {
  language: "en" as const,
  responseStyle: "concise" as const,
  persona: "jarvis" as const,
};

export const mockProjectConfig = {
  ...mockGlobalConfig,
  autoCapture: true,
  checkpointBeforeRisky: true,
  initialized_at: "2025-11-15T00:00:00Z",
};

export const mockVerboseConfig = {
  ...mockGlobalConfig,
  responseStyle: "verbose" as const,
  autoCapture: false,
  checkpointBeforeRisky: false,
};
